### DISCUSSION TITLE
Using xDrip with standalone watch and T-slim pump

### DISCUSSION THREAD

**Discussion by acember** [NightscoutFoundation/xDrip#3812] [General]
Hey, everyone. I'm considering getting the Tandem T-slim pump, but I'm concerned about maintaining the connection with my Nightscout site and my standalone watch (Sony SWR 50). I've heard that a Dexcom transmitter can only be paired with two receivers: right now those are my phone (running xDrip and uploading to NS) and my watch (running xDrip standalone quite often, as I run many miles/hours per week and don't carry my phone). I cannot readily give up either of these -- although phone is the only possible option, provided that a) my data still gets to NS for analysis, either from the watch or from the pump; b) a 'predictive simulation' as good as the one on xDrip for Android will be visible to me on the T-slim. (On the watch, I find it too small to use, but I rely on those simulations heavily). Given these constraints, will it be possible for me to use the T-slim? What would you advise? 

(The 'similar discussion' was indeed similar, but about the Ypsomed pump. I am not at all sure that the same answers would apply to Tandem. If your advice for any reason would be to use a different pump that does closed-loop but isn't Tandem, that advice would also be welcome -- as long as you're not suggesting Medtronic! Currently I'm using a vintage Medtronic Paradigm, sometimes running openAPS.)

**Comment by Navid200** [comment]
There will be a conflict between your watch and t:slim.  

You can consider the following as an alternative approach.

- Get a BlueJay U2 to replace your watch.
- When you are not running: 
  t:slim will use the medical slot of the transmitter.  Phone will use the mobile slot of the transmitter.  Watch will get readings from the phone through Bluetooth.
- When you are running (no phone):
 t:slim will use the medical slot.  Modify a setting on the watch to make it a stand-alone collector using the mobile slot.
 After a run, you change the setting on the watch to make it dependent on the phone again using Bluetooth.  Phone will become the collector using the mobile slot again.

Nightscout:
Only the phone will be set up to upload to Nightscout.  While you are running, there will be no uploads to Nightscout.  When you return from a run and reconnect the transmitter to the phone, xDrip on the phone will backfill up to 3 hours if G6 and up to 24 hours if G7.  After backfill, the backfilled data will be uploaded to Nightscout by the phone.

**Note**
BlueJay U2 is still in beta testing.  You can get one.  But, beta test means still in development.

**Comment by acember** [comment]
Navid, thanks!! That actually sounds like a perfect (to the degree that such things can be perfect) solution. What I'm thinking at the moment is that I'll buy one of these BlueJay watches ahead of the 90-day "returns allowed" period of the T-slim, so that if this doesn't work, I'll know right away and give the pump back.

One small follow-up: I just looked at the BlueJay website. Would you happen to know how serious the "not waterproof" situation is? I have had some quite severe issues with pump buttons from rain and sweat, so gauging whether I would lose CGM data (or worse, permanently ruin my precious watch) would be valuable.

**Reply by Navid200** [reply]
Your CGM data will be in Nightscout and on the transmitter, not on the watch or phone.

I suggest you contact the company and ask.  You cannot rely on what I tell you.

I expect that you are not supposed to go swimming while wearing the watch.
But, a splash of water from rain should be fine.  

I cannot emphasize enough that I am not an employee of the company that sells that watch and have no idea what "not waterproof" means.  You should not buy the watch without consulting with the company about every question you may have about it including whether it is waterproof or not.

**Reply by acember** [reply]
Got it, I completely understand. I thought that perhaps you had this watch yourself and might have some personal experience. Thanks again for your help!
